openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -120,6 +136,7 @@ def _verify_full_account_response(self, response):
         self.assertEqual("world peace", data["goals"])
         self.assertEqual("Park Ave", data['mailing_address'])
         self.assertEqual(self.user.email, data["email"])
+        self.assertTrue(data["is_active"])
```

Comment:
It's interesting that the default value for is_active is True (though when an account is created, it does end up being false, if you look above at AccountSettingsOnCreationTest.